### PR TITLE
Added rule partition_for_dev_shm

### PR DESCRIPTION
**Pull request by rumch-se**
#### Description:

- _A new rule as a part of CIS profile for SLE 12/15_

#### Rationale:

- The rule covers CIS requirement 1.1.6 Ensure /dev/shm is configured (Automated)

### CODE REVIEW

**Comment by dodys** [comment]
I think this item for Ubuntu and RHEL fell out of the CIS benchmark a few months ago as this is handled by systemd tmpfs automatically, if I remember correctly.
@ggbecker @yuumasato maybe you remember more about it.
Maybe this could be reported to CIS to be removed from SUSE's benchmark?

**Comment by yuumasato** [comment]
> I think this item for Ubuntu and RHEL fell out of the CIS benchmark a few months ago as this is handled by systemd tmpfs automatically, if I remember correctly. @ggbecker @yuumasato maybe you remember more about it.

I don't remember. But I don't know if systemd in SLE behaves the same.

> Maybe this could be reported to CIS to be removed from SUSE's benchmark?

This could reported back to CIS, but depends on how it is in SLE12.
Regardless, the CIS for SLE12 has an specific item for `/dev/shm` in `/etc/fstab`.

File: linux_os/guide/system/software/disk_partitioning/partition_for_dev_shm/rule.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+documentation_complete: true
+
+title: 'Ensure /dev/tmp is configured'
```
Comment (marcusburghardt) [inline linux_os/guide/system/software/disk_partitioning/partition_for_dev_shm/rule.yml:3]: It seems this title is not correct.

**Comment by marcusburghardt** [comment]
> > I think this item for Ubuntu and RHEL fell out of the CIS benchmark a few months ago as this is handled by systemd tmpfs automatically, if I remember correctly. @ggbecker @yuumasato maybe you remember more about it.
> 
> I don't remember. But I don't know if systemd in SLE behaves the same.
> 
> > Maybe this could be reported to CIS to be removed from SUSE's benchmark?
> 
> This could reported back to CIS, but depends on how it is in SLE12. Regardless, the CIS for SLE12 has an specific item for `/dev/shm` in `/etc/fstab`.

@rumch-se could you also include the `prodtype` parameter in `rule.yml` and restrict this rule to `sle` products, please?

**Comment by rumch-se** [comment]
Hello @marcusburghardt 
I have done expected corrections
Have a nice day
Rumen

**Review by marcusburghardt — APPROVED** [review]
Thanks @rumch-se

**Comment by marcusburghardt** [comment]
Overriding the CODEOWNERS protection since there is no SUSE contributor with write permission at the moment.

**Pull request merged**